### PR TITLE
Updated the nav bar and header from new design

### PR DESCRIPTION
**Pull request by eugenechantk**
## Work done
- Added nav bar and header
- Added `layout.js` to standardize the nav bar for each page
- Make all elements in page align to left and right margin

## Screenshot
Screenshot 2022-02-28 at 5 12 37 AM

### CODE REVIEW

**Comment by javier123454321** [comment]
Looks good but seems to be pointed to main rather than staging. I think we should point our PRs to staging and only merge to main from staging

**Comment by eugenechantk** [comment]
@javier123454321 updated it to point to staging

File: pages/index.js [hunk 1]
```diff
@@ -4,27 +4,24 @@ import OrgCard from "/components/OrgCard";
...
 import Link from "next/link";
 import Tabs from "/components/Tabs";
+import Hero from '/components/Hero/hero'
+import Nav from '/components/Nav/nav'
+import Layout from "../components/layout";
 
 export default function Home(props) {
   const tabs = ["Military", "Humanitarian"];
   const [openTab, setOpenTab] = useState("Military");
+<<<<<<< HEAD
```
Comment (geositta) [inline pages/index.js:14]: Please remove, thanks

File: pages/index.js [hunk 2]
```diff
@@ -4,27 +4,24 @@ import OrgCard from "/components/OrgCard";
...
   const tabGroup = props.rows.filter((row) =>
     row[4].toLowerCase().includes(openTab.toLowerCase())
   );
+=======
+  const tabGroup = props.rows.filter((row) => row[4].toLowerCase().includes(openTab.toLowerCase()));
+>>>>>>> a7af41478e655711e73e581a4476fc8c62cbe64b
```
Comment (geositta) [inline pages/index.js:20]: 2 more lines to remove, thanks

**Comment by eugenechantk** [comment]
@geositta deleted those lines that you commented on and pull the new changes from staging into this PR

**Pull request merged**